src/utils/allowCorsForAxios.js

```diff
@@ -0,0 +1,5 @@
+const allowCorsForAxios = (axios) => {
+  axios.defaults.headers.common['Access-Control-Allow-Origin'] = '*';
+};
+
+export default allowCorsForAxios;
```

Comment:
Are you using this? If not remove this

Reply:
I am using this in both login and signup pages, I have been taught to write functions with a clear name insted of comments